src/AdminConsole/Components/Shared/BlazorStylesheet.razor

```diff
@@ -0,0 +1,21 @@
+@inject IFileVersionProvider Version
+
+<link rel="stylesheet" href="@Href" />
```

Comment:
@jonashendrickx Remind me, what did we say about the cachebusting? Will we need to fix it in a separate PR?